src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/PackageFeeds/MultiSourcePackageMetadataProvider.cs

```diff
@@ -255,15 +256,34 @@ private async Task<IPackageSearchMetadata> GetPackageMetadataAsync(PackageIdenti
                 tasks.Add(_localRepository.GetPackageMetadataFromLocalSourceAsync(identity, cancellationToken));
             }
 
+            if (_globalLocalRepositories != null)
+            {
+                _globalLocalRepositories.ForEach(x =>
+                    tasks.Add(x.GetPackageMetadataFromLocalSourceAsync(identity, cancellationToken)));
+            }
+
             IEnumerable<IPackageSearchMetadata> completed = (await Task.WhenAll(tasks))
                 .Where(m => m != null);
 
-            IPackageSearchMetadata master = completed.FirstOrDefault(m => !string.IsNullOrEmpty(m.Summary))
-                ?? completed.FirstOrDefault()
-                ?? PackageSearchMetadataBuilder.FromIdentity(identity).Build();
+            PackageSearchMetadataBuilder metadataBuilder = completed.Where(m => !string.IsNullOrEmpty(m.Summary)).Select(PackageSearchMetadataBuilder.FromMetadata).FirstOrDefault()
```

Comment:
Curious what problem warranted this change, which as I can tell, ensures that FromMetadata is called on all `completed` metadata items.

Reply:
I think I just wanted them all to be a ClonedPackageSearchMetadata but seems like that wasn't necessary.